### PR TITLE
fixed issue#1 created a responsive navbar

### LINKED ISSUE #1: Create fixed navbar
## Create a fully responsive sticky navbar
Should contain the following links -
1. Home
2. Events
3. Our Team
4. Feed
5. Contact Us

### PR DESCRIPTION
**Pull request by adnanhashmi09**
Created a responsive navbar which when scrolled to a certain viewport height becomes sticky. Added 3 empty section with height:100vh for testing purposes.

### CODE REVIEW

**Comment by saumyasingh203** [comment]
@adnanhashmi09 
Can you post a screenshot of the same? 
You just have to click on PrtSc and paste it here in the comment box using ctrl+V and it will automatically be uploaded

**Comment by adnanhashmi09** [comment]
image

image

image

image

**Comment by saumyasingh203** [comment]
Great work! @adnanhashmi09  :)
Will merge it

**Comment by adnanhashmi09** [comment]
Thank You :)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+LICENSE
+README.md
```
Comment (snitin315) [inline .gitignore:2]: These files should not be ignored.

Comment (saumyasingh203) [reply .gitignore:2]: Thank you for pointing this out! Will update.

**Review by saumyasingh203 — COMMENTED** [review]
Reviewed and edits incorporated.

**Pull request merged**